src/lava/lib/dl/netx/blocks/process.py

```diff
@@ -0,0 +1,237 @@
+        shape of the block output in (x, y, z) or WHC format.
+    neuron_params : dict
+        dictionary of neuron parameters. Defaults to None.
+    has_graded_input : dict
+        flag for graded spikes at input. Defaults to False.
+    """
+    def __init__(self, **kwargs: Union[dict, tuple, list, int, bool]) -> None:
+        super().__init__(**kwargs)
+        self.shape: tuple = kwargs.pop('shape')
+        self.neuron_params = kwargs.pop('neuron_params', None)
+        self.neuron_process = None
+        if self.neuron_params is not None:
+            self.neuron_process = self.neuron_params.pop('neuron_proc', None)
+        self.neuron = None
+        self.has_graded_input = kwargs.pop('has_graded_input', False)
+        self.has_graded_output = False
+        if 'use_graded_spikes' in self.neuron_params.keys():
+            self.has_graded_output = self.neuron_params['use_graded_spikes']
+
+    def _clean(self) -> None:
+        del self.neuron_params
+        del self.neuron_process
+
+    def _neuron(self, bias: np.ndarray) -> Type[AbstractProcess]:
+        if bias is None:
+            return self.neuron_process(shape=self.shape, **self.neuron_params)
+        else:
+            if len(self.shape) == 1:  # scalar bias value
+                bias_mant = bias
+            elif list(bias.shape) == list(self.shape):
+                # neuron wise bias value
+                bias_mant = bias
+            elif self.shape[-1] == bias.shape[0]:  # channel wise bias value
+                bias_mant = np.zeros(self.shape)
+                bias_mant[:, :] = bias
+            else:
+                raise RuntimeError(
+                    f'Bias of shape {bias.shape} could not be broadcast to '
+                    f'neuron of shape {self.shape}.'
+                )
+            # print(f'{bias=}')
+            # print(f'{bias_mant=}')
+            neuron_params = {'shape': self.shape, **self.neuron_params}
+            neuron_params['bias'] = bias_mant.astype(np.int32)
+            return self.neuron_process(**neuron_params)
+
+    @property
+    def block(self) -> str:
+        return str(self.__class__.__name__)
+
+
+class Input(AbstractBlock):
+    """Input layer block.
+
+    Parameters
+    ----------
+    shape : tuple or list
+        shape of the layer block in (x, y, z)/WHC format.
+    neuron_params : dict
+        dictionary of neuron parameters. Defaults to None.
+    transform : fx pointer or lambda
+        input transform to be applied. Defualts to ``lambda x: x``.
+    bias : np.ndarray or None
+        bias of input neuron. None means no bias. Defaults to None.
+    """
+    def __init__(self, **kwargs: Union[dict, tuple, list, int, bool]) -> None:
+        super().__init__(**kwargs)
+        self.transform = kwargs.pop('transform', lambda x: x)
+
+        if 'bias_exp' not in self.neuron_params:
+            self.neuron_params['bias_exp'] = 6
+
+        self.neuron = self._neuron(kwargs.pop('bias', None))
+
```

Comment:
Do these docstring params need updating?

Reply:
Based on __init__ functionality

Reply:
It's correct.